### PR TITLE
[PCSUP-10440] Supported runtimes for azure serverless defender

### PR DESCRIPTION
**Pull request by avish9**

### CODE REVIEW

**Review by Pubs-MV — COMMENTED** [review]
@avish9 Thank you. Can you please point me to the GH issue or the PCSUP that this change relates to?

**Review by Pubs-MV — COMMENTED** [review]
Is this related to https://github.com/PaloAltoNetworks/prisma-cloud-docs/pull/383

**Comment by Pubs-MV** [comment]
@rcaballeromx please review for any changes to the sys reqs or anywhere else.

**Review by rcaballeromx — APPROVED** [review]

File: compute/admin_guide/install/install_defender/install_serverless_defender.adoc [hunk 1]
```diff
@@ -28,9 +28,10 @@ The following runtimes are supported for AWS Lambda:
 * Python 3.6, 3.7, 3.8, 3.9
 * Ruby 2.7
 
-The following runtimes are supported for Azure Functions:
+The following runtimes are supported for Azure Functions (Windows only):
```
Comment (rcaballeromx) [inline compute/admin_guide/install/install_defender/install_serverless_defender.adoc:31]: @avish9 
Did the support was added for manually embedded Defenders on Windows only or also for Vulnerability and Compliance scanning?

Comment (Pubs-MV) [reply compute/admin_guide/install/install_defender/install_serverless_defender.adoc:31]: ping @avish9

Comment (avish9) [reply compute/admin_guide/install/install_defender/install_serverless_defender.adoc:31]: It's only for Defenders,  it's part of this feature: https://spring.paloaltonetworks.com/twistlock/twistlock/issues/24423

Comment (Pubs-MV) [reply compute/admin_guide/install/install_defender/install_serverless_defender.adoc:31]: @avish9  does this change impact this page https://docs.paloaltonetworks.com/prisma/prisma-cloud/22-06/prisma-cloud-compute-edition-admin/install/system_requirements

Comment (Pubs-MV) [reply compute/admin_guide/install/install_defender/install_serverless_defender.adoc:31]: @rcaballeromx can you follow up on this for sys req updates? I am merging this to publish the correct information in this topic.

**Pull request merged**